modules/apps/commerce/commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/internal/servlet/taglib/CommerceFrontendJsDynamicInclude.java

```diff
@@ -163,6 +164,18 @@ private String _getContent(
 					return commerceOrderCheckoutConfiguration.
 						showSeparateOrderItems();
 				}
+			).put(
```

Comment:
I'm not sure about this, this way this configuration is only accessible via our code, it's not something accessible via an API

Reply:
That was my concern indeed; I was questioning if it is right to have that configuration at instance level, but it's a requirement so it's right.